plugins/org.yakindu.sct.simulation.core/xtend-gen/org/yakindu/sct/simulation/core/util/.gitignore

```diff
@@ -0,0 +1,3 @@
+/.ExecutionContextExtensions.java._trace
+/.ExecutionContextExtensions.xtendbin
+/ExecutionContextExtensions.java
```

Comment:
You should just exclude `/xtend-gen/` :)

Reply:
Yes please ...